codecov/settings_base.py

```diff
@@ -115,6 +115,10 @@
 
 DATABASE_ROUTERS = ["codecov.db.DatabaseRouter"]
 
+# GCS
+GCS_BUCKET_NAME = "codecov"
```

Comment:
I think technically we have a bunch of different buckets (like a separate one for bundle analysis)?
Not sure the organizational scheme we've got going on there to offer an alternative name here, so approved it